doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -166,7 +166,11 @@ The standard Fast Pair API (without extensions) currently supports the :c:func:`
 
 The standard Fast Pair (without extensions) does not require registration of any callback type, meaning all callbacks are optional.
 
-Apart from the callback registration, no additional operations are needed to integrate the standard Fast Pair implementation.
+After the callback registration, the Fast Pair subsystem must be enabled with the :c:func:`bt_fast_pair_enable` function.
+The Fast Pair subsystem readiness can be checked with the :c:func:`bt_fast_pair_is_ready` function.
+The Fast Pair subsystem can be disabled with the :c:func:`bt_fast_pair_disable` function.
+
+Apart from the callback registration and enabling the Fast Pair subsystem, no additional operations are needed to integrate the standard Fast Pair implementation.
```

Comment:
Consider mentioning that user needs to enable Bluetooth and load settings before enabling Fast Pair.

Reply:
`to integrate the core Fast Pair implementation`?

Reply:
> `to integrate the core Fast Pair implementation`?

We have a few mentions (L165, L167) of the standard Fast Pair implementation in this RST file. Calling it here the core Fast Pair implementation would confuse the user. We agreed in previous code reviews to use the "standard" keyword to refer to the core part. I think we should keep it as it is

Reply:
My comment was about keeping the consistency. The confusion here stems from the fact that we use two different words to refer to the same thing.

Is there a `core` use case? Both "core" and "standard" are made-up words by us and are not used in the original Fast Pair specification. When I say that we agreed to the previous wording, I have the following PR in mind:

https://github.com/nrfconnect/sdk-nrf/pull/13229

where there were no comments regarding this wording

I will leave it up to @alstrzebonski to select the approach for this one. However, if we decide to go with the "core" word, then we need to align it consistently in this guide.

Reply:
I will leave it as is. I think it's not confusing and reader can easily understand what we mean.